rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/PostgresqlTableMetaCache.java

```diff
@@ -128,7 +125,13 @@ private TableMeta resultSetMetaToSchema(DatabaseMetaData dbmd, String tableName)
                 col.setColumnDef(rsColumns.getString("COLUMN_DEF"));
                 col.setSqlDataType(rsColumns.getInt("SQL_DATA_TYPE"));
                 col.setSqlDatetimeSub(rsColumns.getInt("SQL_DATETIME_SUB"));
-                col.setCharOctetLength(rsColumns.getInt("CHAR_OCTET_LENGTH"));
+                /**
+                 * @author yanyujie
+                 * @date 2020年6月18日09:24:191
+                 * 为了seata兼容postgresql的charOctetLength这个字段将ColumnMeta改为泛型
+                 * 解决postgresql的驱动兼容性问题
+                 */
```

Comment:
explanation in english.and not write author and date here. add new author only  in add new class  or refactor class.

Reply:
Modified